src/app/pages/home/home.component.html

```diff
@@ -129,11 +131,16 @@ <h3 class="mb-0">Active Requests</h3>
             <tbody *ngFor="let request of requests">
               <tr [ngClass]="{'placed': request.status_name == 'PLACED', 'picked': request.status_name == 'PICKED'}">
                 <th scope="row">
-                  <a [routerLink]="['/edit-profile']">{{request.first_name | titlecase}} {{request.last_name | titlecase}}</a>
+                  <a [routerLink]="['/edit-profile']">{{request.first_name | titlecase}}
+                    {{request.last_name | titlecase}}</a>
                 </th>
-                <td>
-                  <a [title]="request.title" [routerLink]="['/request-detail',request.request_id]">{{ request.title.length > 25 ? (request.title | slice:0:25) + '.....' : request.title}}</a>
+                <td *ngIf="member_role == 'BALEMUYA'">
```

Comment:
Why is this change needed? All we need is to show or hide the request link from the home page.